Sources/Widgets/Widgets3D/ResliceCursorWidget/index.d.ts

```diff
@@ -0,0 +1,107 @@
+import { mat4 } from 'gl-matrix';
+import { Vector2, Vector3 } from '../../../types';
+import { CaptureOn, ViewTypes } from '../../Core/WidgetManager/Constants';
+import vtkAbstractWidgetFactory from '../../Core/AbstractWidgetFactory';
+import vtkPlaneSource from '../../../Filters/Sources/PlaneSource';
+import vtkImageReslice from '../../../Imaging/Core/ImageReslice';
+import vtkImageData from '../../../Common/DataModel/ImageData';
+import vtkRenderer from '../../../Rendering/Core/Renderer';
+
+export interface IDisplayScaleParams {
+  dispHeightFactor: number,
+  cameraPosition: Vector3,
+  cameraDir: Vector3,
+  isParallel: false,
+  rendererPixelDims: Vector2
+}
+
+export interface vtkResliceCursorWidget extends vtkAbstractWidgetFactory {
+
+  /**
+   * @param {ViewTypes} viewType
+   */
+  getRepresentationsForViewType(viewType: ViewTypes): unknown;
+
+  setImage(image: vtkImageData): void;
+
+  setCenter(center: Vector3): void;
+
+  updateCameraPoints(
+    renderer: vtkRenderer,
+    viewType: ViewTypes,
+    resetFocalPoint: boolean,
+    keepCenterFocalDistance: boolean,
+    computeFocalPointOffset: boolean
+  ): void;
+
+  resetCamera(
+    renderer: vtkRenderer,
+    viewType: ViewTypes,
+    resetFocalPoint: boolean,
+    keepCenterFocalDistance: boolean
+  ): void;
+
+
+  getPlaneSource(viewType: ViewTypes): vtkPlaneSource;
+
+  getResliceAxes(viewType: ViewTypes): mat4;
+
+  updateReslicePlane(imageReslice: vtkImageReslice, viewType: ViewTypes): boolean;
+
+  getPlaneSourceFromViewType(type: ViewTypes): vtkPlaneSource;
+
+  getPlaneNormalFromViewType(viewType: ViewTypes): Vector3;
+
+  getOtherPlaneNormals(viewType: ViewTypes): Array<Vector3>;
+
+  getResliceMatrix(): mat4;
+
+  getDisplayScaleParams(): IDisplayScaleParams;
+
+  setScaleInPixels(scale: boolean): void;
+
+  /**
+   * Set the handles visibility for each underlying view widget.
+   * 
+   * @param {Boolean} visible 
+   */
+  setHandleVisiblity(visible: boolean): void
```

Comment:
`setHandleVisibility` is part of vtkAbstractWidgetFactory, so we don't need it here.